app/models/student.rb

```diff
@@ -2,14 +2,24 @@
 #
 # Table name: students
 #
-#  id                  :integer          not null, primary key
-#  created_at          :datetime
-#  updated_at          :datetime
-#  email               :string
-#  confirmation_token  :string
-#  confirmed           :boolean          default(FALSE)
-#  classroom_id        :integer
-#  survey_administered :boolean          default(FALSE)
+#  id                            :integer          not null, primary key
+#  created_at                    :datetime
+#  updated_at                    :datetime
+#  email                         :string
+#  confirmation_token            :string
+#  confirmed                     :boolean          default(FALSE)
+#  classroom_id                  :integer
+#  survey_administered           :boolean          default(FALSE)
+#  fundamentals_quiz_completed   :float            default(-1.0)
```

Comment:
You should probably make a validation for these columns since there are restrictions about what these values can be (eg. -1, or 0 and up, but not -0.5).